src/lib_json/json_value.cpp

```diff
@@ -1170,11 +1170,48 @@ Value const& Value::operator[](CppTL::ConstString const& key) const {
 #endif
 
 Value& Value::append(const Value& value) { return (*this)[size()] = value; }
-
-Value& Value::append(Value&& value) {
-  return (*this)[size()] = std::move(value);
+/// \brief Insert value in array at specific index
+bool Value::insert(ArrayIndex index, const Value& newValue){
+  if(type() != arrayValue){
```

Comment:
Run clang-format on this patch as well. LMK if you have any problems while trying to do that.

Reply:
Finally, I configured the meson/ninja build environment and install clang-format. The result looks pretty good.
```c++
root@dggphicprc07936:~/json/jsoncpp-master/build-shared# meson test --no-rebuild --print-errorlogs
1/2 unittest_jsoncpp_test                   OK       0.02 s
2/2 unittest_jsontestrunner                 OK       0.87 s

Ok:                    2
Expected Fail:         0
Fail:                  0
Unexpected Pass:       0
Skipped:               0
Timeout:               0

```

Reply:
Awesome, thanks for figuring that out!